As of current writing (go 1.13.14), a PIE binary can only be statically linked
when cgo is enabled.

diff --git a/Makefile b/Makefile
index 3225dee..76d46c2 100644
--- a/Makefile
+++ b/Makefile
@@ -23,7 +23,7 @@ lint:
 
 bin/go-md2man: actual_build_flags := $(BUILD_FLAGS) -o bin/go-md2man
 bin/go-md2man: bin
-	@CGO_ENABLED=0 go build $(actual_build_flags)
+	@go build $(actual_build_flags)
 
 bin:
 	@mkdir ./bin
